### ISSUE TITLE
508 Defect 2, Screen Readers, Medical Co-Pays, detail page of individual debt, headings nested improperly.

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#31095]
<h2>Details</h2>
<p>On the detail pages for the individual debt, headings are not properly nested.</p>
<h2>Solution</h2>
The first three headings should be nested as follows:
<ul>
<li>h1 Your copay bill for James A. Haley Veterans’ Hospital (This is already correct but I'm including it for reference)</li>
<li>h2Pay your $30.00 balance or request help before July 2, 2021</li>
<li>h3What if I’ve already requested financial help with my bill?</li>
</ul>
<p>The options under the h3 You can pay your bill in any of these 4 ways should be h4s according to proper heading nesting standards, but in fact don't need to be headings at all. If you want to style them for visual emphasis, I suggest removing the h2 tag and using CSS to style them.</p>
<p>The options under the h2 How do I get financial help for my copays? should be h3s, but again, they do not need to be headings. </p>
<h2>Generally speaking, </h2>
Expandable accordians do not need to be contained within heading tags because they are coded as buttons and so can easily be located by screen reader users. CSS can be used to style them for visual emphasis.
